src/sphinx_book_theme/header_buttons/launch.py

```diff
@@ -41,7 +41,14 @@ def add_launch_buttons(
     # If so, insert the URLs depending on the configuration
     config_theme = app.config["html_theme_options"]
     launch_buttons = config_theme.get("launch_buttons", {})
-    if not launch_buttons or not _is_notebook(app, pagename):
+    if (
+        not launch_buttons
+        or not _is_notebook(app, pagename)
+        or not any(
+            launch_buttons[key]
```

Comment:
Shouldnt we use `launch_buttons.get(key)` in case one is not present so we don't raise a key error?